reports/California.md

```diff
@@ -1008,7 +990,6 @@ id: ca-sacramento-3
 
 * [From perspective of woman pushed, where we see man being choked](https://twitter.com/greg_doucette/status/1268334584443342850)
 * [From a different angle, where we see filming woman pushed down](https://twitter.com/reereeisme65/status/1267378345894789125)
-* [Additional angle](https://www.tiktok.com/@sameoldmatt/video/6833452428025679109)
```

Comment:
Tiny thing - I think this was added to master after your PR - can you include it back?

Reply:
Done.